model/build.gradle

```diff
@@ -5,7 +5,6 @@ compileTestJava.options.compilerArgs.add '-parameters'
 
 dependencies {
     compile("com.google.gwt.inject:gin:$ginVersion")
```

Comment:
Please revert as it was before

Reply:
What exactly should be reverted? Only change I see is the removal of a whitespace which was done automatically whilst pulling the latest version of master.

Reply:
Your file should match what is in master I shouldn't see any changes on this file compared to master.